index.html

```diff
@@ -186,20 +186,20 @@ <h3>
               <p>
 This specification defines the Verifiable Credentials Data Model 2.0 along with
 serializations of that data model. It will replace the current
-<a href="https://www.w3.org/TR/vc-data-model/">VC Data Model 1.0 Recommendation</a>.
+<a href="https://www.w3.org/TR/vc-data-model/">VC Data Model 1.1 Recommendation</a>.
               </p>
 
               <p class="draft-status"><b>Draft state:</b> <a href="https://w3c.github.io/vc-data-model/">Editor's Draft</a> </p>
               <p class="milestone"><b>Expected completion:</b> Q4 2023</p>
               <dl>
                 <dt>
                   <p>
-<b>Adopted Draft:</b> <a href='https://w3c.github.io/vc-data-model/'>Verifiable Credentials Data Model 1.0</a>
+<b>Adopted Draft:</b> <a href='https://w3c.github.io/vc-data-model/'>Verifiable Credentials Data Model 1.1</a>
                   </p>
                   <p>
 <b>Exclusion Draft:</b>
-<a href='https://www.w3.org/TR/2019/REC-vc-data-model-20191119/'>Verifiable Credentials Data Model 1.0</a>
-Exclusion period <b>began</b> 25 July 2019; Exclusion period <b>ended</b> 23 September 2019.
+<a href='https://www.w3.org/TR/2021/REC-vc-data-model-20211109/'>Verifiable Credentials Data Model 1.1</a>
+Exclusion period <b>began</b> 11 November 2021; Exclusion period <b>ended</b> 08 January 2021.
                   </p>
                   <p>
 <b>Other Charter:</b> <a href="https://www.w3.org/2017/vc/WG/charter.html"> https://www.w3.org/2017/vc/WG/charter.html</a>
```

Comment:
By introducing 1.1 (which is fine) we may consider using the current WG charter reference (ie, https://www.w3.org/2020/12/verifiable-credentials-wg-charter.html) or both.

I am not 100% sure which one to take; maybe adding both is the safest bet.

Reply:
I'll use the latest charter ref, that's probably the easiest to understand.

-0.5 to using both, people would probably be confused by that... e.g., "Why are they listing two charters here? Are they both active? Only one of them?"

Reply:
Works for me.

Reply:
Done.